docs/visual/e2e-testing/troubleshooting.md

```diff
@@ -95,3 +95,17 @@ capabilities = {
 ```
 
 Additional information on WebDriver timeouts can be found [here](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Errors/ScriptTimeout).
+
+### cropTo can not crop the target element
+
+#### element is not scrollable
+
+In this case, you could try to use the scrollAndStitchScreenshot approach. For example:
+
+```
+await browser.execute('/*@visual.snapshot*/', 'Home', { scrollAndStitchScreenshot: true });
+```
+
+#### element is scrollable
+
+Unfortunately, we are not supporting this feature for now.
```

Comment:
@xwa130saucer Are there any plans to support this in the near future?

Reply:
Sorry, we don't have plan to support this for now. I moved it to H3 heading as limitation. Does this look good?

Reply:
Yes, looks good. Thanks!